pkgs/tools/system/smartmontools/default.nix

```diff
@@ -26,7 +27,7 @@ in stdenv.mkDerivation rec {
   postPatch = "cp -v ${driverdb} drivedb.h";
 
   configureFlags = [
-    "--with-scriptpath=${stdenv.lib.makeBinPath [ mailutils inetutils ]}"
+    "--with-scriptpath=${lib.makeBinPath ([ inetutils ] ++ lib.optional enableMail mailutils)}"
```

Comment:
Does it even need `inetutils` when not enabling mail support?  I can't think of another reason why smartmontools should talk to the network.

Reply:
The only `inetutils` binary it uses in `update-smart-drivedb` is `ftp`, which has a comment nearby  of `# OpenBSD`. Say the word and I'll make them both optional; I don't know if that single use of `ftp` is worth "breaking" or not (considering it also supports `curl`, `wget`, `svn`, etc).